Source/WebCore/bindings/scripts/IDLAttributes.json

```diff
@@ -396,6 +396,9 @@
             "contextsAllowed": ["type"],
             "values": ["*"]
         },
+        "IgnoreSuspension": {
```

Comment:
I’m not entirely sure that our terminology is ideal. There may be a better term for it that could be easier to understand in future. One example of what I mean is we could have named this `BypassDocumentActiveChecks`.

Reply:
Or even better, use terminology that matches how the specification describes it. I read the specification text and now would use the name `BypassDocumentFullyActiveCheck`.